packages/core/android/src/main/kotlin/com/datadog/reactnative/DdSdkBridgeExt.kt

```diff
@@ -11,7 +11,8 @@ import com.facebook.react.bridge.WritableNativeMap
 
 internal fun List<*>.toWritableArray(): WritableNativeArray {
     val list = WritableNativeArray()
-    forEach {
+    for (it in iterator()) {
+        @Suppress("NotImplementedDeclaration")
```

Comment:
what is the reason for this declaration?

Reply:
This is because we have a `else -> TODO()` at the end of the `when`.